app/presenters/worldwide_organisation_presenter.rb

```diff
@@ -89,16 +91,18 @@ def main_office
     )
   end
 
-  WorldwideOffice = Struct.new(:contact, :has_access_and_opening_times?, :public_url, keyword_init: true)
-
   def home_page_offices
     return [] unless content_item.dig("details", "home_page_office_parts")
 
     content_item.dig("details", "home_page_office_parts").map { |office|
       contact = linked_contact(office["contact_content_id"])
       next unless contact
 
-      WorldwideOrganisation::LinkedContactPresenter.new(contact)
+      WorldwideOffice.new(
```

Comment:
Could this (and the declaration of `WorldwideOffice`) be moved into a moved, to avoid having the same code repeated in both the `main_office` and `home_page_offices` methods?